docs/tutorials/02_create_your_first_app/03_adding-variables-and-methods.md

```diff
@@ -10,7 +10,7 @@ Now let's start adding some code to our backend!
 We will use Motoko programming language. You can get a quick overview of Motoko at
 the [Motoko Language Tour](/motoko/intro/index.md).
 
-We will edit file `src/poll_backend/main.mo`.
+In this tutorial, we will edit file `src/poll_backend/main.mo`, but if you want to play and iterate with Motoko, the [Motoko Playground](https://m7sm4-2iaaa-aaaab-qabra-cai.ic0.app) is a good web-based option.
```

Comment:
Should this be icp0.io?

Reply:
No, good question. It’s a legacy canister